workflows/raredisease.nf

```diff
@@ -35,81 +35,42 @@ ch_multiqc_custom_config = params.multiqc_config ? Channel.fromPath(params.multi
 ========================================================================================
 */
 
-// Don't overwrite global params.modules, create a copy instead and use that within the main script.
-def modules = params.modules.clone()
-
-// Switch for saving references
-if (!params.save_reference) {
-    modules['bwa_mem2_index'].publish_files = false
-    modules['samtools_faidx'].publish_files = false
-}
-
 //
 // SUBWORKFLOW: Consisting of a mix of local and nf-core/modules
 //
-include { INPUT_CHECK } from '../subworkflows/local/input_check' addParams( options: [:] )
-include { CHECK_VCF } from '../subworkflows/local/prepare_vcf' addParams( options: [:] )
+include { INPUT_CHECK } from '../subworkflows/local/input_check'
+include { CHECK_VCF } from '../subworkflows/local/prepare_vcf' //addParams( options: [:] )
 
 /*
 ========================================================================================
     IMPORT NF-CORE MODULES/SUBWORKFLOWS
 ========================================================================================
 */
 
-def multiqc_options   = modules['multiqc']
-multiqc_options.args += params.multiqc_title ? Utils.joinModuleArgs(["--title \"$params.multiqc_title\""]) : ''
-
 //
 // MODULE: Installed directly from nf-core/modules
 //
-include { FASTQC } from '../modules/nf-core/modules/fastqc/main'  addParams( options: modules['fastqc'] )
-include { MULTIQC } from '../modules/nf-core/modules/multiqc/main' addParams( options: multiqc_options )
-include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/modules/custom/dumpsoftwareversions/main' addParams( options: [publish_files : ['_versions.yml':'']] )
 
+include { FASTQC  } from '../modules/nf-core/modules/fastqc/main'   //addParams( options: modules['fastqc'] )
+include { MULTIQC } from '../modules/nf-core/modules/multiqc/main'  //addParams( options: multiqc_options )
+include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/modules/custom/dumpsoftwareversions/main'  //addParams( options: [publish_files : ['_versions.yml':'']] )
```

Comment:
Can't we remove these comment ?

Reply:
Yes absolutely. It was removed on my branch 🤔 . @ramprasadn ?

Reply:
That's odd, because I just checked the code base as of the point I took over and I see these comments there. I wonder if there is a conflict between what we have online and what you have locally @projectoriented ?